### PR TITLE
HARP-5671: 3D Label Placement. [DO NOT REVIEW]

### PR DESCRIPTION
**Pull request by amsgames94**

### CODE REVIEW

File: @here/harp-debug-datasource/lib/DebugTileDataSource.ts [hunk 1]
```diff
@@ -49,38 +52,64 @@ export class DebugTile extends Tile {
...
+            const material = new THREE.MeshBasicMaterial({
+                color: "red",
+                wireframe: true,
+                transparent: true,
+                depthTest: false
+            });
+            const mesh = new THREE.Mesh(g, material);
+            mesh.name = `debug_tile_${this.tileKey.toHereTile()}`;
+            mesh.renderOrder = 100000;
+            this.objects.push(mesh);
```
Comment (ninok) [inline @here/harp-debug-datasource/lib/DebugTileDataSource.ts]: The text is missing in the sphere projection case

File: @here/harp-examples/src/globe.ts [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+ */
+
+import { EarthConstants, GeoCoordinates, sphereProjection } from "@here/harp-geoutils";
+import { CopyrightElementHandler, CopyrightInfo, MapView } from "@here/harp-mapview";
+import { APIFormat, OmvDataSource } from "@here/harp-omv-datasource";
+import { accessToken } from "../config";
+
+import * as THREE from "three";
+
+export namespace GlobeExample {
```
Comment (ninok) [inline @here/harp-examples/src/globe.ts]: Why do we have the example in this PR?

File: @here/harp-mapview/lib/MapView.ts [hunk 3]
```diff
@@ -739,8 +749,12 @@ export class MapView extends THREE.EventDispatcher {
             0.1,
             4000000
         );
+        this.m_camera.up.set(0, 0, 1);
         this.m_lookAtDistance = 0;
         this.m_focalLength = 0;
+        this.m_scene.add(this.m_camera); // ensure the camera is added to the scene.
+
+        this.m_rtcCamera = new THREE.PerspectiveCamera(); // this should not be added to the scene.
```
Comment (ninok) [inline @here/harp-mapview/lib/MapView.ts]: Why is this change here? It is defenitely out of scope for this change.

**Comment by amsgames94** [comment]
@ninok Sorry for the confusion, but this change is in a bit of a proof of concept state to validate some of the ideas being implemented in #453 . Will add ``[DO NOT REVIEW]`` message until this PR is merged.

**Comment by amsgames94** [comment]
@heremaps-bot retest this please.

**Comment by amsgames94** [comment]
Development of these changes will continue in https://github.com/heremaps/harp.gl/pull/453.

**Pull request closed without merging**